src/index.ts

```diff
@@ -39,7 +40,7 @@ export async function build({
     }
   } else {
     if (!(await ensureLocalPhp())) {
-      console.log(`
+      debug(`
```

Comment:
Should a message like this appear only when `NOW_BUILDER_DEBUG` is defined? I mean, it is pretty essential that the user sees this. It is not like a debug message that you might want to see ;).